### PR TITLE
fix: incorrect selected status in wallet status bar

### PR DESCRIPTION
**Pull request by nuanyang233**
## Description

Closes #MF-1648

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Review by Randolph314 — DISMISSED** [review]

File: packages/mask/src/utils/components/WalletStatusBar/WalletMenuItem.tsx [hunk 1]
```diff
@@ -55,8 +66,16 @@ export const WalletMenuItem = memo<WalletMenuItemProps>(
...
                 <ListItemIcon>
-                    <Radio checked={selected} />
+                    {selected ? (
+                        <Icons.RadioButtonChecked
+                            className={classes.icon}
+                            style={{ filter: 'drop-shadow(0px 0px 6px rgba(28, 104, 243, 0.6))' }}
+                        />
+                    ) : (
+                        <Icons.RadioButtonUnChecked className={classes.icon} />
+                    )}
```
Comment (UncleBill) [inline packages/mask/src/utils/components/WalletStatusBar/WalletMenuItem.tsx]: Why not just control with size?

```suggestion
                        <Icons.RadioButtonChecked
                            size={24}
                            style={{ filter: 'drop-shadow(0px 0px 6px rgba(28, 104, 243, 0.6))' }}
                        />
                    ) : (
                        <Icons.RadioButtonUnChecked size={24} />
                    )}
```

Comment (nuanyang233) [reply packages/mask/src/utils/components/WalletStatusBar/WalletMenuItem.tsx:78]: There are two different icons.

File: packages/mask/src/plugins/Avatar/Application/NFTAvatarsDialog.tsx [hunk 2]
```diff
@@ -66,6 +70,8 @@ export function NFTAvatarDialog(props: NFTAvatarsDialogProps) {
         props.onClose()
     }, [props.onClose])
 
+    useEffect(() => setSelectedAccount(account || wallets?.[0]?.identity || ''), [account, wallets])
```
Comment (UncleBill) [inline packages/mask/src/plugins/Avatar/Application/NFTAvatarsDialog.tsx]:
```suggestion
    useEffect(() => setSelectedAccount(account || wallets?.[0]?.identity || ''), [account, wallets?.[0]?.identity])
```

**Review by guanbinrui — DISMISSED** [review]

**Review by guanbinrui — APPROVED** [review]

**Comment by guanbinrui** [comment]
@nuanyang233 Please resolve the conflicts.

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**